cmd/kmd/main.go

```diff
@@ -36,30 +38,43 @@ const (
 	kmdLogFilePerm = 0640
 )
 
-func main() {
-	dataDir := flag.String("d", "", "kmd data directory")
-	timeoutSecs := flag.Uint("t", 0, "number of seconds after which to kill kmd if there are no requests. 0 means no timeout.")
-	flag.Parse()
+var (
+	dataDir     string
+	timeoutSecs uint64
+)
+
+func init() {
+	kmdCmd.Flags().StringVarP(&dataDir, "data-dir", "d", "", "KMD data directory.")
+	kmdCmd.Flags().Uint64VarP(&timeoutSecs, "timout-secs", "t", 0, "Number of seconds that KMD will run for before termination.")
+}
 
+var kmdCmd = &cobra.Command{
+	Use:   "kmd",
+	Short: "Key Management Daemon (KMD)",
+	Long:  `The Key Management Daemon (KMD) is a low level wallet and participation
+key management tool. It works in conjunction with algod and goal to keep
+secrets safe. An optional timeout flag will automatically terminate KMD
+after a number of seconds has elapsed, allowing a simple way to ensure KMD
+will be shutdown in a timely manner. This is a blocking command that.`,
```

Comment:
typo: "This is a blocking command that."
nit: kmd doesn't manage participation keys
nit: can you please replace all instances of KMD with kmd?